### DISCUSSION TITLE
how do you add a loading screen

### DISCUSSION THREAD

**Discussion by NK-523** [KadeArchive/Kade-Engine#2186] [Q&A] [answered]
How do you add a custom loading screen like in the VS tricky mod.

**Comment by BoloVEVO** [comment, marked as answer]
Go to to caching.hx around line 70 and edit kadelogo path image If your image is big like tricky one, modify kadelogo.x and kadelogo.y to your dimensions.

**Reply by NK-523** [reply]
This worked, thanks!
